contrib/acpica/update.py

```diff
@@ -0,0 +1,2 @@
+# the tag versioning scheme over at https://github.com/acpica/acpica is bogus
+ignore = True
```

Comment:
it actually isn't, you can map the RMM_DD_YY values back to YYYYMMDD

..whether you should is another story

Reply:
that solution would break in 2100 though

Reply:
that's a problem for future generations

Reply:
IMO defining `fetch_versions` to remap RMM_DD_YY to YYYYMMDD is better than outright ignoring, because that way `update-check` still works.